src/meta/src/hummock/manager/mod.rs

```diff
@@ -1958,6 +1957,54 @@ where
                 }),
             );
     }
+
+    #[named]
+    pub async fn start_lsm_stat_report(hummock_manager: Arc<Self>) -> (JoinHandle<()>, Sender<()>) {
+        use crate::hummock::model::CompactionGroup;
+        let (shutdown_tx, mut shutdown_rx) = tokio::sync::oneshot::channel();
+        let join_handle = tokio::spawn(async move {
+            let mut min_interval = tokio::time::interval(std::time::Duration::from_secs(10));
+            loop {
+                tokio::select! {
+                    // Wait for interval
+                    _ = min_interval.tick() => {
+                    },
+                    // Shutdown
+                    _ = &mut shutdown_rx => {
+                        tracing::info!("Lsm stat reporter is stopped");
+                        return;
+                    }
+                }
+
+                let id_to_config = hummock_manager.get_compaction_group_map().await;
+                let current_version = {
+                    let mut versioning_guard =
+                        write_lock!(hummock_manager.as_ref(), versioning).await;
+                    versioning_guard.deref_mut().current_version.clone()
+                };
+
+                let compaction_group_ids_from_version = get_compaction_group_ids(&current_version);
+                let default_config = CompactionConfigBuilder::new().build();
```

Comment:
The `default_config` should be the one from CompactionGroupManager::default_config.